fs/config/config.go

```diff
@@ -58,6 +58,8 @@ type Config struct {
 	MountTimeoutSec                int64  `toml:"mount_timeout_sec"`
 	FuseMetricsEmitWaitDurationSec int64  `toml:"fuse_metrics_emit_wait_duration_sec"`
 
+	// RetryableHTTPClientConfig is the complete config for a retryable http client
+	RetryableHTTPClientConfig `toml:"http"`
```

Comment:
nit: do we want to change `http` to `rhttp` to make it more clear?

Reply:
I stuck with `http`, I think its seems a little more understandable to the user than `rhttp`. Also the `http.retries` implies that the client is "retryable".